server/helpers/helperFunction.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable no-param-reassign */

```

Comment:
could you explain to me where you are using this function and why you decided to make this a helper function

Reply:
I am using this to log in because login is still using data structure. I will delete that file after I implement the database on login.